CONTRIBUTING.md

```diff
@@ -1,3 +1,51 @@
 # Contributing Guide
 
-See our Brigade project [Contributing Guide](https://github.com/brigadecore/community/blob/main/contributing.md).
+Brigade SDK for JavaScript is an official library of the Brigade project and as
+such follows all of the policies laid out in the main
+[Brigade Contributor Guide](https://docs.brigade.sh/topics/contributor-guide/).
+Anyone interested in contributing to this gateway should familiarize themselves
```

Comment:
```suggestion
Anyone interested in contributing to this repository should familiarize themselves
```

Reply:
(Or `project`, dealer's choice.  Applies to similar comments on the other applicable PRs)